source/customizing_look_and_feel/04_configuring_data_visualizations/basemaps.rst

```diff
@@ -1,39 +1,166 @@
 
-.. image:: images/basemaps__mapbox-newyork.png
-   :alt: Data over Mapbox
+A basic basemap is used by default for all maps of all OpenDataSoft domains. It is however possible to change that default basemap, as well as to make other basemaps available in an OpenDataSoft portal. It will allow users to change the basemap at any time on a map, and even to change the default basemap of a specific dataset.
 
-If you don't have a Mapbox account, you can create one for free, although it is going to have a small number of allowed
-"map views". See `Mapbox plans <https://www.mapbox.com/plans/>`_ for more information.
+.. admonition:: Note
+   :class: note
 
-Once you are logged on the Mapbox website with your account, you can go on
-`your account page <https://www.mapbox.com/account/apps/>`_ and copy your Default Public Token or create a new Public
-access token; this is the key that will be used by OpenDataSoft to display maps on the behalf of your own Mapbox
-account.
+   To know more about basemaps and how to use them, you can read our blog post "`What is a basemap & how to choose the best one for your data <https://www.opendatasoft.com/2015/09/03/what-is-a-basemap-and-how-to-choose-the-best-one-for-your-data/>`_".
 
-Go to *Look & feel*, then *Data visualizations* ; you will find a Mapbox configuration form where you can
-enter your Mapbox public access token. A message will confirm whether your access token is valid or not.
+To manage basemaps, go to Look & feel > Data visualizations. In this subsection of the back office menu, look for the "Basemaps" area.
 
 .. ifconfig:: language == 'en'
 
-    .. image:: images/basemaps__mapbox-integration--en.png
-       :alt: Mapbox integration configuration
+    .. image:: images/basemaps_overview--en.png
+       :alt: Overview of the Basemaps area in the Look and feel > Data visualizations section.
 
-.. ifconfig:: language == 'fr'
+Basemaps should be managed from the table of the "Basemaps" area. This table displays the currently available basemaps that can be used on the domain. It comprises 5 columns:
+
+- |icon-reorder| to change the order of the basemaps. The first one is the default one on the whole domain.
+- Label: to name the basemaps. Each basemap should have one label per language available on the domain.
+- Basemap: to choose the basemap.
+- Preview: to see what the basemaps look like.
+- |icon-trash| to delete a basemap.
+
+
+Adding default basemaps
+-----------------------
+
+By default, OpenDataSoft provides several basemaps. They are the default basemaps displayed in the drop-down menu of the Basemap table column (described in the introduction of this documentation). All these default basemaps can be made available on the platform, using the table. Even though the basemaps appear in the list of the default basemaps, they will only be usable once added to the table.
```

Comment:
"Even though the basemaps appear in the list of ~the~ default basemaps"

Reply:
"of the Basemap**s** table column" ?

Reply:
I used the actual name of the column in the interface, which is "Basemap" and not "Basemaps".